.rubocop.yml

```diff
@@ -12,30 +12,149 @@
 AllCops:
   NewCops: enable
   TargetRubyVersion: 3.0
+  Exclude:
+  - 'lib/docopt.rb'
+
+Security/Eval:
+  Enabled: false
 
 Layout/HashAlignment:
-  EnforcedHashRocketStyle:
-    - separator
-  EnforcedColonStyle:
-    - separator
```

Comment:
Could we please change this back? This enforces fixing up the URL hashes which I have been using for most of the last two years...

Reply:
This will get changed back once this pr gets merged-- applying this across the whole codebase required enough manual changes that it is annoying to have to re-apply them by hand every time I rebase due to merge conflicts. This won't undo any of the pre-existing formatted files using this rule, btw.

Reply:
Applying automated changes across the whole codebase is ideal and probably possible with a few tweaks to the rules.

Reply:
Most of the changes are automated, but some require manual readjustment of the code. I've temporarily disables these rules, and once this PR gets merged I'll re-enable them in subsequent PR's.